docs/scripts/amplify_wav.md

```diff
@@ -0,0 +1,44 @@
+---
+tags:
+    - audio
+    - wav
+    - saymore
```

Comment:
is this script saymore-specific?

Reply:
No, it just amplifies parts of WAV files that it thinks contain speech. I add the amplified files to SayMore and transcribe them that way but you could do something else.

Reply:
Should I remove the tag?

Reply:
I went ahead and did it -- my idea for tags was to make them software-specific. I see that `audio` is a good tag, but if a script is not tailored to a particular software, don't include it.